src/api/liveSpecsExt.ts

```diff
@@ -351,17 +347,14 @@ export interface LiveSpecsExtQuery_ByLiveSpecId {
     spec_type: Entity;
     last_pub_id: string;
     spec: any;
+    connector_id: string;
```

Comment:
Need this so we can pass it along during hydration (instead of passing it in the URL)